worker/uniter/manifold.go

```diff
@@ -41,6 +41,7 @@ type Logger interface {
 // Manifold will depend.
 type ManifoldConfig struct {
 	AgentName             string
+	ModelType             model.ModelType
```

Comment:
Ah... I see now. Did we make sure that the other places we specify ManifoldConfig have a valid value here?

Reply:
Yeah, previously we only the IaaS uniter was configured using this now we have k8sagent unit.
CAAS operator configures uniter in a different way.